content/vault/v1.19.x/content/docs/secrets/databases/oracle.mdx

```diff
@@ -158,6 +158,70 @@ you will need to enable ipc_lock capabilities for the plugin binary.
 
    See the [Commands](/vault/docs/commands#files) docs for more details.
 
+### Rootless Configuration and Password Rotation for Static Roles
+
+<EnterpriseAlert product="vault"/>
+
+The Oracle database plugin supports using Static Roles and its password rotation mechanisms with a Rootless
+DB connection configuration. In this workflow, a static DB user can be onboarded onto Vault's static role rotation
+mechanism without the need of privileged root accounts to configure the connection. Instead of using a single root
+connection, multiple dedicated connections to the DB are made for each static role. This workflow does not support
+dynamic roles/credentials.
+
+~> Note: It is **highly recommended** that the DB users being onboarded as static roles
+have the minimum set of privileges. Each static role will open a new connection into the DB.
+Granting minimum privileges to the DB users being onboarded ensures that multiple
+highly-privileged connections to an external system are not being made.
+
+~> Note: Out-of-band password rotations will cause Vault to be out of sync with the state of
+the DB user, and will require manually updating the user's password in the external Oracle
+DB in order to resolve any errors encountered during rotation.
+
+1.  Enable the database secrets engine if it is not already enabled:
+
+```shell-session
+$ vault secrets enable database
+Success! Enabled the database secrets engine at: database/
+```
+
+By default, the secrets engine will enable at the name of the engine. To
+enable the secrets engine at a different path, use the `-path` argument.
```

Comment:
```suggestion
1.  Enable the database secrets engine if it is not already enabled:

    ```shell-session
    $ vault secrets enable database
    Success! Enabled the database secrets engine at: database/
    ```

    By default, the secrets engine will enable at the name of the engine. To
    enable the secrets engine at a different path, use the `-path` argument.
```
Formatting correction: indent all content related to a particular step, including code examples